tests/src/main/scala/scalaz/tests/IOTests.scala

```diff
@@ -0,0 +1,107 @@
+package scalaz
+package tests
+
+import scala.{ List, Range, StringContext }
+import scala.Predef.String
+
+import Scalaz._
+
+import testz._, z._
+import laws._, FunctorLaws._
+
+import zio._
+
+object IOTests {
+
+  case class Error(msg: String)
+
+  // a probably-too-large selection of IO actions; more could be added
+  val ios = {
+    type IOES = IO[Error, String]
+    type Endo = IOES => IOES
+    val seeds = List[IOES](
+      IO.now("now"),
+      IO.unit.map(_ => "unit"),
+      IO.point("point"),
+      IO.fail(Error("error")),
+      //IO.sleep(scala.concurrent.duration.Duration.fromNanos(1)).map(_ => "slept")
+    )
```

Comment:
This is good *for now* but we should probably have some tests that the laws work w.r.t `sync` and `async`, which will require more specific IO actions.